app/src/main/java/org/openobservatory/ooniprobe/activity/MeasurementDetailActivity.java

```diff
@@ -119,6 +124,15 @@ public static Intent newIntent(Context context, int id) {
 				.replace(R.id.body, detail)
 				.replace(R.id.head, head)
 				.commit();
+		snackbar = Snackbar.make(coordinatorLayout, R.string.Snackbar_ResultsNotUploaded_Text, Snackbar.LENGTH_INDEFINITE).setAction(R.string.Snackbar_ResultsNotUploaded_Upload, v1 -> new MKCollectorResubmitSettingsAsyncTask(this).execute(null, measurement.id));
```

Comment:
This line needs better indentation.

Perhaps we should setup a linter that automatically indents at some reasonable column width to make our life easier :)

Reply:
done. for global linter i suggest to pospone

Reply:
https://github.com/ooni/probe-android/issues/219